ui/lib/core/addon/components/masked-input.hbs

```diff
@@ -32,7 +32,7 @@
       aria-label={{or @name "masked input"}}
       {{on "change" this.onChange}}
       {{on "keyup" (fn this.handleKeyUp @name)}}
-      data-test-textarea
+      data-test-textarea={{or @name ""}}
```

Comment:
I think we don't need or if the default is an empty string
```suggestion
      data-test-textarea={{@name}}
```

Reply:
I thought that as well, but without the conditional it broke any test selector that didn't pass in a name. I suspect because it comes in as undefined?